### ISSUE TITLE
"chrome_binary" doesn't work in nodeconfig file

### ISSUE THREAD

**Issue by sohammukherjee** [groupon/Selenium-Grid-Extras#115]
We have provided the location of chrome_binary in nodeconfig file but the chrome browser still doesn't work. 
Is there a way to sort this thing out???

Please find the attached image of my nodeconfig file....

![image](https://cloud.githubusercontent.com/assets/8623750/7181562/54a95758-e469-11e4-9a10-1063726657e5.png)

**Comment by smccarthy** [comment]
What doesn't work? Does it throw an exception? Is there any message in the logs? I did a quick search and found : https://github.com/SeleniumHQ/selenium/issues/342 (which was fixed pretty recently). Could that be the issue you are seeing? If so, maybe it will be fixed in the next selenium release I assume.

If that isn't your issue, can you let us know what error you are seeing?

Thanks!

**Comment by TeqBuk** [comment]
Hi smccarthy, 

This is Mahendra Rathod on behalf of Soham. We have created our own selenium grid using Selenium Grid Extras where the browser binaries are installed in different locations on nodes. We have found that when we provide firefox_binary path, the selenium grid extras is able to pickup correct firefox binary. But when we provide chrome_binary, it does not pick up. 

We are also getting the same error log mentioned in : SeleniumHQ/selenium#342

Thanks,
Mahendra Rathod.
